apps/web/src/views/FixedStaking/hooks/useHandleWithdrawSubmission.ts

```diff
@@ -25,7 +25,23 @@ export function useHandleWithdrawSubmission({
   const { t } = useTranslation()
   const { toastSuccess, toastInfo } = useToast()
   const { callWithGasPrice } = useCallWithGasPrice()
-  const { fetchWithCatchTxError, loading: pendingTx } = useCatchTxError()
+  const { fetchWithCatchTxError, loading: pendingTx } = useCatchTxError({
+    throwCustomeError: () => {
+      const linkElement = createElement(
```

Comment:
If we change the file extension to `tsx` then we can make use of the jsx markup instead of calling `createElement`?